src/auth/frappe.auth.controller.ts

```diff
@@ -408,6 +409,19 @@ export default class FrappeAuthController extends AuthController {
       : RequestResponse.fail(this.handleError("logout", r.error));
   }
 
+  public async setSessionStatusInfo({
+    sessionStatusInfo
+  }: {
+    sessionStatusInfo: SessionStatusInfo;
+  }) {
+    if (isNodeJS()) {
```

Comment:
I believe this can be made common whether it's on the server or client (browser). Is there a specific reason why it's specific to NodeJS

Reply:
Nope no reason and yes i agree.